lib/rate-limiter.js

```diff
@@ -12,13 +12,14 @@ class RateLimiter {
    *
    * @param {Object} options
    */
-  constructor (options) {
+  constructor(options) {
```

Comment:
Please follow the code style for this project, which requires a space before function parentheses.

Do you have Prettier or another formatter installed on your system? Please deactivate it for hapi-rate-limitor and follow the ESLint styling rules of the project